.devcontainer/Dockerfile

```
@@ -0,0 +1,29 @@
+ARG VARIANT=latest
+FROM oven/bun:${VARIANT}
+
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)